packages/ui/tailwind.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { Config } from "tailwindcss";
+
```

Comment:
Create new project for Tailwind.

Reply:
I'm done with it. But I'm unable to squash the commits due to git merge in between the commits